src/utils/selector.ts

```diff
@@ -280,9 +283,16 @@ export class Selector {
     private static async promptForInput(defs: PromptVariableDefinition[]): Promise<Map<string, string> | null> {
         const promptVariables = new Map<string, string>();
         for (const { name, description } of defs) {
+            // In name resembles some kind of password prompt, enable password InputBox option
+            const passwordPromptNames = ['password', 'Password', 'PASSWORD', 'passwd', 'Passwd', 'PASSWD', 'pass', 'Pass', 'PASS'];
```

Comment:
I would add 'pwd', 'Pwd', 'PWD'

And maybe we should use case-insensitive comparison here?